packages/fleet/charts/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v2
+name: fleet
+version: v0.2.0
+appVersion: v0.2.0
+icon: https://raw.githubusercontent.com/StrongMonkey/fleet/master/fleet.svg
+description: Managing fleets of Kubernetes clusters
```

Comment:
This needs to be a better description. Something to indicate gitops workflow, I think